components/create_comment/create_comment.jsx

```diff
@@ -391,7 +391,8 @@ export default class CreateComment extends React.PureComponent {
     handleSubmit = async (e) => {
         e.preventDefault();
 
-        if (this.props.isTimezoneEnabled) {
+        const containsAtChannelMsg = containsAtChannel(this.state.draft.message);
```

Comment:
Should we check all the conditions below (e.g. `enableConfirmNotificationsToChannel` and the members count? Otherwise, we're still invoking this async call even though we would never display the modal.

Reply:
Maybe just move this inside the existing checks and before the call to `showNotifyAllModal`?

Reply:
sure, sounds good

Reply:
addressed your feedback, thanks for your review and comments.